core/chains/evm/client/node.go

```diff
@@ -453,6 +457,15 @@ func (n *node) HeaderByNumber(ctx context.Context, number *big.Int) (header *typ
 }
 
 func (n *node) SendTransaction(ctx context.Context, tx *types.Transaction) error {
+	var rpcDomain string
+	start := time.Now()
+	defer func() {
+		duration := time.Since(start)
+		promEVMPoolSendTransactionTiming.
+			WithLabelValues(n.chainID.String(), n.name, rpcDomain, "false").
+			Observe(float64(duration))
+	}()
```

Comment:
nit/ could tighten scope a bit here:
```suggestion
	defer func(start time.Time) {
		duration := time.Since(start)
		promEVMPoolSendTransactionTiming.
			WithLabelValues(n.chainID.String(), n.name, rpcDomain, "false").
			Observe(float64(duration))
	}(time.Now())
```

Reply:
Yup good point, done.